### PR TITLE
Updated FireFox Documentation

### PR DESCRIPTION
**Pull request by Unthrottled**
Related to this comment https://github.com/doki-theme/doki-theme-web/pull/83#pullrequestreview-840113633

### CODE REVIEW

**Review by ZimCodes — COMMENTED** [review]
Changes are good! However with the update for the new keywords `all` & `device`, I would like to update the enums for these terms to stay consistent. Is it possible for you to open this pr for edit?

File: README.md [hunk 1]
```diff
@@ -156,6 +160,54 @@ There is a know issue with session restoration on re-open though. [See GitHub is
...
+**Match Device**
+
+`device` will load only themes based on your current device theme settings *(either light or dark)*.
+In other words, your device settings determines what characters are present in the popup menu.
+
+#### Random Select *(Dark System Setting)*
+https://user-images.githubusercontent.com/23222943/146148772-c7e236ed-6be6-46b6-ac2d-aa3f1b1326ca.mp4
+
+#### Mixed Mode *(Light System Setting)*
+https://user-images.githubusercontent.com/23222943/146149663-22ad428e-0cc7-45f8-9700-1fc4c6f616d2.mp4
```
Comment (ZimCodes) [inline README.md:182]: I think it would be better to show a visual demo of the feature as well.

File: README.md [hunk 2]
```diff
@@ -156,6 +160,54 @@ There is a know issue with session restoration on re-open though. [See GitHub is
...
+
+`druthers` is similar to `device` except the character chosen for the light theme and the
+character chosen for the dark theme will alternate depending on your current device settings.
+
+For clarity, if I choose *Asuna* for the **light theme** & *Hatsune Miku* for the **dark theme**.
+If the device setting is **currently dark**, then the **dark** *Hatsune Miku theme* will show. If the device
+setting is *light*, then the **light** *Asuna* theme will show.
+
+#### Example (Maple *(light)* / Jahy *(dark)*)
+https://user-images.githubusercontent.com/23222943/146766085-ecb1b416-a678-4cbc-a75c-44c7fb5b32de.mp4
```
Comment (ZimCodes) [inline README.md:194]: Another visual example conveying the `druthers` feature.

**Review by ZimCodes — APPROVED** [review]
Alright. I was able to change the keywords `system` & `default` to match the new keywords `device` & `all`. The previous changes have been review as well.

File: README.md (hunk 1 above)
Comment (Unthrottled) [reply README.md:182]: Yeah, I think that is a good idea!

I was just thinking, some of the examples showcase a UI that was outdated. However, I think that keeping these all up to date with how it looks as of the current release is too much work.

File: README.md (hunk 2 above)
Comment (Unthrottled) [reply README.md]: Yup, that's legit. I did change the documentation to reflect the content's of the video. Felt a bit weird to give a written example & a visual example that where different.

**Comment by Unthrottled** [comment]
Oh yeah, by the way. If you need help updating your get set up, just let me know. 

I would prefer to have a linear git history, with nothing too crazy going on. 
Not against having a bunch of commits merged into master, just as long as they are linear, which is why I've been squashing stuff recently (cause there's a lot of stuff going on the forked branch you've been working on).

**Comment by Unthrottled** [comment]
I'll assume you are good with the changes that I pushed after your approval, if not (whoops, I'll wait next time :)

**Pull request merged**